app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -318,11 +323,29 @@
 
   /* Step 2 */
   .life-stages {
+    .all-stages-wrapper {
+      margin-left: auto;
+      margin-right: auto;
+      @media #{$small-only} {
+        width: 360px;
+      }
+      @media #{$medium-only} {
+        width: 540px;
+      }
+      @media #{$large-only} {
+        width: 720px;
+      }
+      @media #{$xlarge-up} {
+        width: 900px;
+      }
+    }
+
     .stage-wrapper {
       display: inline-block;
       overflow: hidden;
-      height: 142px;
       margin-bottom: 3%;
+      height: 150px;
+      width: 180px;
```

Comment:
Yes, I had a reason but maybe its not a good one:

The icons can either be: a percentage width to fill the div, in which case we can't maintain the circular aspect ratio because the height doesn't like to auto-adjust to match the width. This is why the icons were rectangular up until now. Or, we could maintain the right aspect ratio but the icons would float left.

My solution to maintain aspect ratio and have the icons float center was to wrap all the icons into a div of set width with auto-margins (for centering). That div would change width by the right amount (180px) with each breakpoint in order to change the number of icon columns without having float left issues.

Let me know if there is a better way :)